src/components/Header.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
we don't need to import `react` anymore since the introduction of JSX. If you're curious, you can read up more about this here:
https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj

Reply:
Ahh, cool!